.goreleaser.yml

```
@@ -20,6 +20,10 @@ builds:
         goarch: arm
     env:
       - CGO_ENABLED=0
+    ldflags:
+      - -s -w -X github.com/controlplaneio/kubesec/v2/cmd.version={{.Version}}
+      - -s -w -X github.com/controlplaneio/kubesec/v2/cmd.commit={{.FullCommit}}
+      - -s -w -X github.com/controlplaneio/kubesec/v2/cmd.date={{.Date}}
```

`.Date` is not reproducible, we can use the date of the commit AKA `SOURCE_DATE_EPOCH` in RFC 3339 format instead

```suggestion
    ldflags:
      - -s -w
      - -X "github.com/controlplaneio/kubesec/v2/cmd.version={{.Version}}"
      - -X "github.com/controlplaneio/kubesec/v2/cmd.commit={{.FullCommit}}"
      - -X "github.com/controlplaneio/kubesec/v2/cmd.date={{.CommitDate}}"
```